test/e2e/reconfigure_test.go

```diff
@@ -0,0 +1,40 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/types"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	nc "github.com/openshift/windows-machine-config-operator/pkg/controller/windowsmachine/nodeconfig"
+)
+
+// reconfigurationTest tests that the correct behavior occurs when a previously configured Machine is configured
+// again. In practice this should not happen, however it simulates a scenario where a Machine was almost completely
+// configured, an error occurred, and the Machine is requeued. This is a scenario that should be expected to be ran into
+// often enough.
```

Comment:
`This is a scenario that should be expected to be ran into often enough.`

Is this true? I would rather say `This is a scenario that is expected to happen often enough to warrant this test..` But what confuses me is the previous sentence state this should not happen in practice.

Reply:
I reworded it to be more readable